xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -1389,33 +1389,27 @@ bool CPVRGUIInfo::GetRadioRDSBool(const CFileItem *item, const CGUIInfo &info, b
 
 namespace
 {
-  int TimeFromDateTime(time_t datetime)
+  std::string TimeToTimeString(time_t datetime, TIME_FORMAT format, bool withSeconds)
   {
     CDateTime time;
     time.SetFromUTCDateTime(datetime);
-    return time.GetHour() * 60 * 60 + time.GetMinute() * 60 + time.GetSecond();
+    return time.GetAsLocalizedTime(format, withSeconds);
   }
 } // unnamed namespace
 
 void CPVRGUIInfo::CharInfoTimeshiftStartTime(TIME_FORMAT format, std::string &strValue) const
 {
-  if (format == TIME_FORMAT_GUESS)
```

Comment:
Has this logic been moved somewhere or was it obsolete?

Reply:
Obsolete. Will be done by CDateTime::GetAsLocalizedTime(string, bool) which  will be called from CDateTime::GetAsLocalizedTime(TIME_FORMAT, bool)